server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -726,7 +726,9 @@ public void apply(Settings value, Settings current, Settings previous) {
             TelemetrySettings.TRACER_SAMPLER_PROBABILITY,
             TelemetrySettings.METRICS_PUBLISH_INTERVAL_SETTING,
             TelemetrySettings.TRACER_FEATURE_ENABLED_SETTING,
-            TelemetrySettings.METRICS_FEATURE_ENABLED_SETTING
+            TelemetrySettings.METRICS_FEATURE_ENABLED_SETTING,
+            TelemetrySettings.TRACER_SAMPLER_ACTION_PROBABILITY,
+            TelemetrySettings.TRACER_SPAN_SAMPLER_CLASSES
```

Comment:
Please remove these setting from here, please use `OTelTelemetrySettings` (they are specific to OTEL)

Reply:
Did added it in TelemetrySettings other sampling probability was also added under the same (like `TRACER_SAMPLER_PROBABILITY`)

Reply:
`TRACER_SPAN_SAMPLER_CLASSES` & co are static settings (not a cluster wide settings) that are configured per Otel plugin

Reply:
Agreed TRACER_SPAN_SAMPLER_CLASSES, should go to OtelTelemetrySettings.